src/lib/components/paginationInline.svelte

```diff
@@ -17,18 +17,15 @@
     function handleOptionClick(page: number) {
         if (currentPage !== page) {
             offset = limit * (page - 1);
-            currentPage = page;
```

Comment:
was removed, but default reactive logic is different, right?

Reply:
it was never needed because of the `$:` syntax. the new linter complains about this

Reply:
yeah, but the logic on `$:` and manual assignment is diff no.? that yields same result?

Reply:
it actually triggers it twice 👍🏻  one change with changing offset and then the re-assignment